src/ipc/ipc4/handler.c

```diff
@@ -402,13 +404,17 @@ static int ipc_wait_for_compound_msg(void)
 	int try_count = 30; /* timeout out is 30 x 10ms so 300ms for IPC */
 
 	while (atomic_read(&msg_data.delayed_reply)) {
+		pm_policy_state_lock_get(PM_STATE_ACTIVE, 1);
```

Comment:
please specify the meaning of "1", other sof code use macro like: PM_ALL_SUBSTATES instead of direct numbers.

Reply:
Will be done better on upstream.